lib/config.js

```diff
@@ -1,8 +1,13 @@
 'use strict'
+
 let options
 const config = {}
 const corePlugin = require('./plugin.js')
 
+function sort (arr) {
```

Comment:
I decided to add a new function since this is used twice; feel free to rename it later to sortAscending or sortDescending (I always mix these two up 😛)